src/parser/warlock/affliction/modules/talents/SoulConduit.js

```diff
@@ -39,11 +41,24 @@ class SoulConduit extends Analyzer {
     const avgDamage = this._totalUAdamage / (this._totalTicks > 0 ? this._totalTicks : 1);
     const shardsGained = this.soulShardTracker.getGeneratedBySpell(SPELLS.SOUL_CONDUIT_SHARD_GEN.id);
     const estimatedUAdamage = shardsGained * TICKS_PER_UA * avgDamage;
+    const totalSpent = this.soulShardTracker.spent;
+    // since we want to get the amount of shards we would MOST LIKELY get, we intentionally need to try higher values for k than usually possible
+    const { partial: probabilities } = binomialCDF(Math.round(totalSpent / 2), totalSpent, SC_PROC_CHANCE);
```

Comment:
so if i understand this right, you're calling it with a reasonably high value just to get it to compute the first `Math.round(totalSpent / 2)` entries in `partial`?

Reply:
Not sure if I understood you right, I'm calling CDF (even though I don't need it), because in `partial` I get PMF values for `k` values lower than `Math.round(totalSpent / 2)`. Since it's a binomial distribution, the pattern looks roughly like this 

![image](https://user-images.githubusercontent.com/5068584/50798591-2fc2a780-12d9-11e9-95e7-5d827e28eee6.png)

and I just want to know, at which `k` the probability is the highest, the peak of the function. Same goes for Poisson binomial distribution, for fun I charted it in Excel and it looks roughly the same.

Reply:
sorry, i wasn't clear, but i figured it out by talking with you on discord (the question was specifically why `Math.round(totalSpent / 2)` and not just `totalSpent`).